go/vumitools/api.py

```diff
@@ -585,6 +584,21 @@ def from_config_async(cls, config, command_publisher=None,
         sender = AsyncMessageSender(command_publisher)
         returnValue(cls(manager, redis, sender, metric_publisher))
 
+    def get_batch_manager(self):
+        from vumi_message_store.message_store import MessageStoreBatchManager
```

Comment:
I did it that way because I wanted to limit the scope of things that touched the message store while I was juggling both versions, but that's not really relevant anymore.

Longer-term, I'm not sure access to the message store (especially the operational message store) really belongs in `VumiApi` and I'd like to avoid exposing it in places I don't have to.